packages/wdio-spec-reporter/src/index.js

```diff
@@ -387,18 +387,29 @@ class SpecReporter extends WDIOReporter {
     getEnviromentCombo (caps, verbose = true, isMultiremote = false) {
         const device = caps.deviceName
         const browser = isMultiremote ? 'MultiremoteBrowser' : (caps.browserName || caps.browser)
-        const version = caps.version || caps.platformVersion || caps.browser_version
-        const platform = caps.os ? (caps.os + ' ' + caps.os_version) : (caps.platform || caps.platformName)
+        /**
+         * fallback to different capability types:
+         * browserVersion: W3C format
+         * version: JSONWP format
+         * platformVersion: mobile format
+         * browser_version: invalid BS capability
+         */
+        const version = caps.version || caps.browserVersion || caps.platformVersion || caps.browser_version
```

Comment:
platform is looking good but version should be changed to:
```js
const version =  caps.browserVersion || caps.version || caps.platformVersion || caps.browser_version
```
the reason for this is so that we keep the preferences in the same order. after that, things seem good! :)